### ISSUE TITLE
question about convalution layer

### ISSUE THREAD

**Issue by shouldnotfail** [DAA233/generative-inpainting-pytorch#27]
Hi DAA233, I found some differences between your code and Jiahuiyu's code, my question is, why at end of each gen_conv layer in Jiahuiyu's code tensor will be divided into two parts and two different activation functions will be used, do you have any idea about it? because it seems like that you just remove this part
 `  x, y = tf.split(x, 2, 3);
    x = activation(x)
    y = tf.nn.sigmoid(y)
    x = x * y
    return x`

**Comment by DAA233** [comment]
Hi @shouldnotfail, sorry for the late response.

This repo is a PyTorch implementation of just DeepFillv1, not including DeepFillv2. Refers to Jiayu's DeepFillv1 code [here](https://github.com/JiahuiYu/generative_inpainting/tree/v1.0.0).
